eng/common/scripts/Verify-Links.ps1

```diff
@@ -115,12 +117,14 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 function CheckLink ([System.Uri]$linkUri)
 {
   if ($checkedLinks.ContainsKey($linkUri)) { return }
```

Comment:
This should now return `$checkedLinks[$linkUri]`.

Reply:
The CheckLink function does not return anything. Do we want to change it to return boolean?